examples/xgboost_baseline.py

```diff
@@ -42,7 +50,13 @@
 dfs: Dict[str, pd.DataFrame] = {}
 entity_table = dataset.db.table_dict[task.entity_table]
 entity_df = entity_table.df
-col_to_stype = infer_df_stype(entity_df)
+
+if args.dataset in stype_dict:
+    col_to_stype = stype_dict[args.dataset][task.entity_table]
```

Comment:
Can we actually set `col_to_stype` for stackex as well? That's more robust and avoids repeated computation.